### PR TITLE
zh-cn: update the translation of HTTP vary

### PR DESCRIPTION
**Pull request by zjffun**
Original: https://github.com/mdn/content/blob/1368da1927b8d414e1c43a52142b9317f73cfce7/files/en-us/web/http/headers/vary/index.md

Detail: https://dochub.zjffun.com/translate/mdn-zh-cn/mdn/translated-content/main/files/zh-cn/web/http/headers/vary/index.md

### CODE REVIEW

File: files/zh-cn/web/http/headers/vary/index.md [hunk 1]
```diff
@@ -45,12 +44,12 @@ Vary: User-Agent
...
 
-## 更多
+## 参见
 
-- {{HTTPHeader("Cache-Control")}}
-- {{HTTPHeader("User-Agent")}}
-- [Best Practices for Using the Vary Header – fastly.com](https://www.fastly.com/blog/best-practices-for-using-the-vary-header)
+- [理解 Vary 标头 - Smashing Magazine](https://www.smashingmagazine.com/2017/11/understanding-vary-header/)
+- [使用 Vary 标头的最佳实践 – fastly.com](https://www.fastly.com/blog/best-practices-using-vary-header)
+- [内容协商](/en-US/docs/Web/HTTP/Content_negotiation)
```
[the remark this answers is not in the document]
Comment (zjffun) [reply files/zh-cn/web/http/headers/vary/index.md:55]: Great!

File: files/zh-cn/web/http/headers/vary/index.md [hunk 2]
```diff
@@ -5,37 +5,36 @@ slug: Web/HTTP/Headers/Vary
...
 {{HTTPSidebar}}
 
-**`Vary`** 是一个 HTTP 响应头部信息，它决定了对于未来的一个请求头，应该用一个缓存的回复 (response) 还是向源服务器请求一个新的回复。它被服务器用来表明在 [content negotiation](/zh-CN/docs/Web/HTTP/Content_negotiation) algorithm（内容协商算法）中选择一个资源代表的时候应该使用哪些头部信息（headers）.
-
-在响应状态码为 {{HTTPStatus("304")}} `Not Modified` 的响应中，也要设置 Vary 首部，而且要与相应的 {{HTTPStatus("200")}} `OK` 响应设置得一模一样。
-
-| Header type                           | {{Glossary("Response header")}} |
-| ------------------------------------- | ------------------------------- |
-| {{Glossary("Forbidden header name")}} | no                              |
+**`Vary`** HTTP 响应标头描述了除方法和 URL 之外影响响应内容的请求消息。大多数情况下，这用于在使用[内容协商](/zh-CN/docs/Web/HTTP/Content_negotiation) 时创建缓存键。
```
Comment (jasonren0403) [inline files/zh-cn/web/http/headers/vary/index.md:8]:
```suggestion
**`Vary`** HTTP 响应标头描述了除方法和 URL 之外影响响应内容的请求消息。大多数情况下，这用于在使用[内容协商](/zh-CN/docs/Web/HTTP/Content_negotiation)时创建缓存键。
```

File: files/zh-cn/web/http/headers/vary/index.md [hunk 3]
```diff
@@ -45,12 +44,12 @@ Vary: User-Agent
 
 {{Compat}}
 
-## 更多关于浏览器兼容性：
+### 兼容性备注
 
-- [Vary with care – Vary header problems in IE6-9](https://blogs.msdn.microsoft.com/ieinternals/2009/06/17/vary-with-care/)
+- [小心使用 Vary – Vary 标头在 IE6-9 的问题](https://blogs.msdn.microsoft.com/ieinternals/2009/06/17/vary-with-care/)
```
Comment (jasonren0403) [inline files/zh-cn/web/http/headers/vary/index.md:49]:
```suggestion
- [小心使用 Vary – Vary 标头在 IE6-9 的问题](https://docs.microsoft.com/archive/blogs/ieinternals/vary-with-care)
```

**Review by jasonren0403 — COMMENTED** [review]
一些建议

File: files/zh-cn/web/http/headers/vary/index.md (hunk 2 above)
Comment (zjffun) [reply files/zh-cn/web/http/headers/vary/index.md:8]: Thank you.

**Review by jasonren0403 — APPROVED** [review]
Thanks @zjffun 👍

**Pull request merged**